Makefile

```diff
@@ -73,7 +73,7 @@ else
 			HOST_ARCH = x86
 		endif
 		ifeq ($(shell uname -m),armv7l)
-			HOST_ARCH = armv7l
+			HOST_ARCH = armhf
```

Comment:
I think "generate the same armhf package" is an OS-specific assumption. E.g. just because `armv7hf` actually maps to `armhf` on Debian, there might be separate `armv6hf` and `armv7hf` packages on some other OS. So IMHO we should keep the `Makefile` as specific as possible, and then let `architecture-convert.sh` map that to whatever arch-strings different systems / platforms / OSes / tools need, as necessary.